generated/ruby/test/test_errorreporting.rb

```diff
@@ -0,0 +1,101 @@
+#
+# Integration tests for vkit generated client library for Google Cloud ErrorReporting APIs.
+#
+
+PROJECT_NAME = "gapic-test".freeze
+SERVICE = "default".freeze
+VERSION = "20160920t135819".freeze
+USER_NAME = "testuser".freeze
+#
+# Integration test for ReportErrorsServiceApi.report_error_event
+#
+# Report first two error events (to be one group) and third event (to be another group)
+require "google/cloud/error_reporting/v1beta1/report_errors_service_api"
+require "google/devtools/clouderrorreporting/v1beta1/report_errors_service_pb"
+ReportErrorsServiceApi = Google::Cloud::ErrorReporting::V1beta1::ReportErrorsServiceApi
+ReportedErrorEvent = Google::Devtools::Clouderrorreporting::V1beta1::ReportedErrorEvent
+ServiceContext = Google::Devtools::Clouderrorreporting::V1beta1::ServiceContext
+ErrorContext = Google::Devtools::Clouderrorreporting::V1beta1::ErrorContext
+HttpRequestContext = Google::Devtools::Clouderrorreporting::V1beta1::HttpRequestContext
+SourceLocation = Google::Devtools::Clouderrorreporting::V1beta1::SourceLocation
```

Comment:
I'm not sure how much those lines help the code. Could it be
```ruby
V1beta1 = Google::Devtools::Clouderrorreporting::V1beta1
```

and those classes can be referred like `V1beta1::ServiceContext`, and I think that's not that bad.